src/extras/primitives/registerPrimitive.js

```diff
@@ -75,6 +69,33 @@ module.exports = function registerPrimitive (name, definition) {
         }
       },
 
+      applyDefaultComponents: {
+        value: function () {
+          var self = this;
+          var defaultData = utils.extendDeep(this.defaultAttributes);
```

Comment:
OK, I think since I removed keeping track of component data, this is no longer needed. Good catch.